tests/visualization_tests/test_optimization_history.py

```diff
@@ -184,3 +206,23 @@ def fail_objective(_: Trial) -> float:
 
     figure = plot_optimization_history(studies, error_bar=True)
     assert len(figure.data) == 0
+
+
+@pytest.mark.parametrize("direction", ["minimize", "maximize"])
+def test_error_bar_in_optimization_history(direction: str) -> None:
+    def objective(trial: Trial) -> float:
+        return trial.suggest_float("x", 0, 1)
+
+    studies = [create_study(direction=direction) for _ in range(3)]
+    suggested_params = [0.1, 0.3, 0.2]
+    for x, study in zip(suggested_params, studies):
+        study.enqueue_trial({"x": x})
+        study.optimize(objective, n_trials=1)
+    figure = plot_optimization_history(studies, error_bar=True)
+
+    mean = np.mean(suggested_params)
+    std = np.std(suggested_params)
+
+    assert_almost_equal(figure.data[0].y, mean)
+    assert_almost_equal(figure.data[2].y, mean + std)
+    assert_almost_equal(figure.data[3].y, mean - std)
```

Comment:
A nit, but how about applying the same function call style as `np.array_equal`? 
```suggestion
    np.assert_almost_equal(figure.data[0].y, mean)
    np.assert_almost_equal(figure.data[2].y, mean + std)
    np.assert_almost_equal(figure.data[3].y, mean - std)
```

Reply:
Thanks! Done: https://github.com/optuna/optuna/pull/3234/commits/0beaab4ae3f4d42c413ca92cdd1eab46fdcac70e

Reply:
Wow! Thank you for your quick fix!